interfaces/kmod/backend_test.go

```diff
@@ -137,12 +116,11 @@ func (s *backendSuite) TestRemovingSnapRemovesModulesConf(c *C) {
 
 func (s *backendSuite) TestSecurityIsStable(c *C) {
 	// NOTE: Hand out a permanent snippet so that .conf file is generated.
-	s.Iface.PermanentSlotSnippetCallback = func(slot *interfaces.Slot, securitySystem interfaces.SecuritySystem) ([]byte, error) {
-		if securitySystem == interfaces.SecurityKMod {
-			return []byte("module1\nmodule2"), nil
-		}
-		return nil, nil
+	s.Iface.KModPermanentSlotCallback = func(spec *kmod.Specification, slot *interfaces.Slot) error {
+		spec.AddModule("module1")
```

Comment:
It's similar to the earlier case, although in this case I agree it makes sense to have 2 modules returned as I'm testing if security is stable, multiple modules make a batter test case.